src/tools/jedutil.cpp

```diff
@@ -2167,9 +2206,22 @@ static pal_data paldata[] = {
 		config_pal20x10_pins,
 		nullptr,
 		nullptr},
-	/*{"PAL22V10", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},
-	{"GAL20V8A", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},
+	/*{"GAL20V8A", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},
```

Comment:
Why is the GAL20V8A removed?

Reply:
The GAL20V8A is part of a comment.  In C /* signifies the start of a comment block and */ signifies the end.  I just moved the start of the comment from PAL22V10 to GAL20V8A since the PAL22V10 device is now supported.